### ISSUE TITLE
Dual Mining DCR and ETH

### ISSUE THREAD

**Issue by OdinSon23** [nanopool/Claymore-Dual-Miner#113]
So I've been messing with this for the better part of a day and can't get it to work right. My mine files looks like this: 

./ethdcrminer64 -epool eth-us-west1.nanopool.org:9999 -ewal WALLET ADDRESS -epsw x 
#$ -dpool dcr.suprnova.cc:3252 -dwal spaceexplorer23.Odinson -dpsw Might -allpools 1  -dcri 16

I have the DCR line commented out at the moment because I can't get it to work. 
At first I couldn't get it to even connect to the supernova pool no matter what, in dual mode it would connect to a pascal pool. (which, different issue, I'd like to dabble with pascal, but it is a bitch to set up). 

in order to make it connect to the supernova pool I had to do this to the dpools.txt file:

POOL: dcr.suprnova.cc:3252, WALLET:YOUR_WALLET.YOUR_PAYMENTID.YOUR_WORKER/YOUR_EMAIL, PSW: x, WORKER: , ESM: 0, ALLPOOLS: 0
#POOL: pasc-eu2.nanopool.org:15555, WALLET: YOUR_WALLET.YOUR_PAYMENTID.YOUR_WORKER/YOUR_EMAIL, PSW: x, WORKER: , ESM: 0, ALLPOOLS: 0
#POOL: pasc-us-east1.nanopool.org:15555, WALLET: YOUR_WALLET.YOUR_PAYMENTID.YOUR_WORKER/YOUR_EMAIL, PSW: x, WORKER: , ESM: 0, ALLPOOLS: 0
#POOL: pasc-us-west1.nanopool.org:15555, WALLET: YOUR_WALLET.YOUR_PAYMENTID.YOUR_WORKER/YOUR_EMAIL, PSW: x, WORKER: , ESM: 0, ALLPOOLS: 0
#POOL: pasc-asia1.nanopool.org:15555, WALLET: YOUR_WALLET.YOUR_PAYMENTID.YOUR_WORKER/YOUR_EMAIL, PSW: x, WORKER: , ESM: 0, ALLPOOLS: 0

basically added supernova, and commented out all others. 

HOWEVER, even though I can connect to the supernova pool now....every share I mined when I had it running was rejected. And it said I wasn't authorized on the dcr side. 

Any help would be appreciated. I'm using claymore 9.5 at the moment.

**Comment by lutsifer** [comment]
You know...in the beginning I had pretty much same issue.  For the second coin I could connect only to one pool (DCR one i think) and it was barely working. SIA? PASCAL? Blah..
Then I red somewhere that the routers sometimes filter out this traffic as suspicious:)
It was hard to find where exactly but when I found it, now I can mine whatever I wish - they all run normally.

**Comment by OdinSon23** [comment]
DCR: Authorization failed
: (null)

I get this no matter what....but I'm almost 100% sure I'm using the right stuff for the supernova pool.

**Comment by ADSC17** [comment]
@OdinSon23 I'm going to assume your new to this? Originally I was in the same boat as you with however it turns out with supernova and coinmine.pl your wallet and worker goes like this;

EthDcrMiner64.exe -epool eu1.ethermine.org:14444 -ewal <wallet>.<worker> -epsw x -dpool dcr-eu.coinmine.pl:2222 -dwal <username>.<worker> -dpsw x -platform 2 -allpools 1

I hope this helps :). Also to note you need to assign your wallet to your username in the edit account section to direct funds if you decide to use auto payout

**Comment by douglashb** [comment]
@ADSC17 Thank you so much, you are my hero, work fine!

**Comment by KonfuciousK** [comment]
Thank you ADSC17, fixed the the issue perfectly.
